### DISCUSSION TITLE
Webserver/UI based config variable definitions?

### DISCUSSION THREAD

**Discussion by aejeffor** [gtxaspec/wz_mini_hacks#472] [Q&A]
I ran across a table somewhere of the configuration variables that you are able to edit from the webserver UI camera config pages.  I've scoured the documentation in the zip file, but can't seem to place where I read them.  Most of the variables are self-explanatory, but on my floodlight cam, I've got a handful that aren't as straightforward.  I'm hoping there's a reference doc that maps these all out?

**Comment by akivacp** [comment]
Some of the settings in the web browser has a question mark you can hover your mouse over.

**Reply by ejeffords** [reply]
yeah, i can see those, but there are many without these tooltips that don't naturally point to the name of any of the GUI settings in the app.
